.travis.yml

```diff
@@ -2,9 +2,9 @@
 sudo: required
 
 # whitelist
-branches:
-  only:
-    - master
+#branches:
```

Comment:
Not sure I understand why the whitelist is commented out.

Reply:
I'm going to put it back before I merge. But I needed to get rid of it to get Travis to build the branch. Not sure why tho